openscenario/openscenario_interpreter/include/openscenario_interpreter/simulator_core.hpp

```diff
@@ -66,10 +68,39 @@ class SimulatorCore
   class CoordinateSystemConversion
   {
   protected:
+    static auto checkRoutingAlgorithm(const RoutingAlgorithm::value_type & routing_algorithm)
```

Comment:
Copy passing is faster for objects smaller than word size.
```suggestion
    static auto checkRoutingAlgorithm(RoutingAlgorithm::value_type routing_algorithm)
```

Reply:
Thank you, I've overlooked that it is an enum type.